src/platform/monitoring/DowntimeNotification/util/helpers.js

```diff
@@ -26,6 +27,29 @@ export function getStatusForTimeframe(startTime, endTime) {
   return externalServiceStatus.ok;
 }
 
+export function createGlobalMaintenanceWindow({
+  startTime,
+  endTime,
+  externalServices = defaultExternalServices,
+}) {
+  return [
+    {
+      attributes: {
+        externalService: 'global',
+        startTime,
+        endTime,
+      },
+    },
+    ...Object.keys(externalServices).map(externalService => ({
+      attributes: {
+        externalService,
+        startTime,
+        endTime,
+      },
+    })),
+  ];
```

Comment:
Nit-picky, but I feel like this is something that should happen in state/reducer. As in - the action creator dispatches just the `global` downtime, and it's translated into state as downtime for all services. But conceptually, it would be cool if wasn't necessary to expand global downtime into each service (and so any consumer of downtime notifications always has `global` as a dependency) but I see why we're doing it this way.

Reply:
yeah good redux practice is to put everything you can into the reducer. expanding it in the reducer makes sense.